docs/tutorials/quick-start/index.rst

```diff
@@ -0,0 +1,645 @@
+
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: example-tls
+      namespace: foo
+    data:
+      tls.crt: <base64 encoded cert>
+      tls.key: <base64 encoded key>
+    type: kubernetes.io/tls
+
+It can take a minute or two for the cloud provider to provide and link a public
+IP address. When it is complete, you can see the external IP address using the
+``kubectl`` command:
+
+.. code-block:: shell
+  :emphasize-lines: 4
+
+  kubectl get svc
+  NAME                                       TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                      AGE
+  kubernetes                                 ClusterIP      10.63.240.1     <none>           443/TCP                      23m
+  quickstart-nginx-ingress-controller        LoadBalancer   10.63.248.177   35.233.154.161   80:31345/TCP,443:31376/TCP   16m
+  quickstart-nginx-ingress-default-backend   ClusterIP      10.63.250.234   <none>           80/TCP                       16m
+
+This command shows you all the services in your cluster (in the ``default``
+namespace), and any external IP addresses they have. When you first create the
+controller, your cloud provider won't have assigned and allocated an IP address
+through the LoadBalancer yet. Until it does, the external IP address for the
+service will be listed as ``<pending>``.
+
+Your cloud provider may have options for reserving an IP address prior to
+creating the ingress controller and using that IP address rather than assigning
+an IP address from a pool. Read through the documentation from your cloud
+provider on how to arrange that.
+
+Step 3 - Assign a DNS name
+==========================
+
+The external IP that is allocated to the ingress-controller is the IP to which
+all incoming traffic should be routed. To enable this, add it to a DNS zone you
+control, for example as `example.your-domain.com`.
+
+This quickstart assumes you know how to assign a DNS entry to an IP address and
+will do so.
+
+Step 2 - Deploy an Example Service
+==================================
+
+Your service may have its own chart, or you may be deploying it directly with
+manifests. This quickstart uses manifests to create and expose a sample service.
+The example service uses `kuard`_, a demo application which makes an excellent
+back-end for examples. 
+
+The quickstart example uses three manifests for the sample. The first two are a
+sample deployment and an associated service:
+
+- deployment manifest: `deployment.yaml`_
+
+.. literalinclude:: example/deployment.yaml
+   :language: yaml
+
+- service manifest: `service.yaml`_
+
+.. literalinclude:: example/service.yaml
+   :language: yaml
+
+.. _`deployment.yaml`: https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/deployment.yaml
+.. _`service.yaml`: https://raw.githubusercontent.com/jetstack/cert-manager/master/docs/tutorials/quick-start/example/service.yaml
```

Comment:
Ah, I've just realised that you are doing that already and these are just links to the actual source 🙈 ignore!

Reply:
It's a fair point though - I'm referencing the files in GitHub at your source control. I thought this might be OK and a nice acceleration, but I can just as easily reference these in the docs directly. The pathing will get a bit odd to read, but I could step someone through "clone this repo" to reference these files as they sit in the docs directory.